app/src/main/java/org/mifos/mobilebanking/utils/DateHelper.java

```diff
@@ -26,11 +27,11 @@
 
     /**
      * the result string uses the list given in a reverse order ([x, y, z] results in "z y x")
-     *
      * @param integersOfDate [year-month-day] (ex [2016, 4, 14])
      * @return date in the format day month year (ex 14 Apr 2016)
      */
     public static String getDateAsString(List<Integer> integersOfDate) {
+        if (integersOfDate.size() != 3) return ""; // no data entry in database for the client
```

Comment:
No need to add check here, Please remove it. Where ever you are trying to parse this date. use if lese there 

```
if (integersOfDate.size() != 3) {
// No DOB Found 
} else {
// Show DOB after parsing.
}
```

Reply:
okay.